### DISCUSSION TITLE
Unable to setup the project

### DISCUSSION THREAD

**Discussion by srikanth4kakumanu** [givanz/Vvveb#11] [General]
Hi Team,

  We have downloaded the project from this link "https://docs.vvveb.com/." After downloading, we have placed the project in Tomcat under webapps folder with the name ROOT & tried to run the Server.

When tried to run the project, it is showing as 404 Not found. Is this because of not having index.html file in the Project?

Can you please help us in this setup.

**Comment by givanz** [comment]
Hi

Vvveb CMS is not written in java but php, you need to setup a webserver with php to run Vvveb CMS, the webserver should load index.php from the root folder.
If you don't want to manually install and configure apache/nginx and php with mysql you can use  [Xampp](https://www.apachefriends.org/download.html) or [Wampp](https://sourceforge.net/projects/wampserver/)  

You can also find nginx.conf for [nginx](https://docs.vvveb.com/nginx) and .htaccess for  [apache](https://docs.vvveb.com/apache) in the root folder if you configure your webserver manually.
